src/scss/vendors/_lineup.scss

```
@@ -23,4 +23,41 @@ $lu_include_font_awesome: false !default;
+// sort neutral
+.lu-action-sort::before {
+    background-color: currentColor;
+    content: '11' !important; /* This is necessary for the icons to have the same size */
+    -webkit-mask-image: url("data:image/svg+xml,%3Csvg width='512' height='472' fill='inherit' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M8.20093 110.314C-2.73364 121.248 -2.73364 138.977 8.20093 149.912C19.1355 160.846 36.864 160.846 47.7986 149.912L101.402 96.308V443.711C101.402 459.175 113.938 471.711 129.402 471.711C144.866 471.711 157.402 459.175 157.402 443.711V96.2896L211.024 149.912C221.959 160.846 239.687 160.846 250.622 149.912C261.556 138.977 261.556 121.248 250.622 110.314L153.246 12.9376C152.213 11.3649 151.005 9.87317 149.622 8.49023C144.05 2.91797 136.713 0.185425 129.41 0.292358C122.108 0.185913 114.772 2.91858 109.201 8.49036C107.818 9.87317 106.61 11.3646 105.577 12.9373L8.20093 110.314ZM354.579 375.692V28.2893C354.579 12.8253 367.114 0.289307 382.579 0.289307C398.043 0.289307 410.579 12.8253 410.579 28.2893V375.71L464.201 322.088C475.135 311.154 492.864 311.154 503.799 322.088C514.733 333.023 514.733 350.752 503.799 361.686L406.423 459.062C405.389 460.635 404.181 462.127 402.799 463.51C397.226 469.082 389.889 471.815 382.587 471.708C375.285 471.814 367.949 469.081 362.378 463.51C360.995 462.127 359.787 460.635 358.754 459.063L261.378 361.686C250.443 350.752 250.443 333.023 261.378 322.088C272.312 311.154 290.041 311.154 300.975 322.088L354.579 375.692Z' /%3E%3C/svg%3E");
+    mask-image: url("data:image/svg+xml,%3Csvg width='512' height='472' fill='inherit' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M8.20093 110.314C-2.73364 121.248 -2.73364 138.977 8.20093 149.912C19.1355 160.846 36.864 160.846 47.7986 149.912L101.402 96.308V443.711C101.402 459.175 113.938 471.711 129.402 471.711C144.866 471.711 157.402 459.175 157.402 443.711V96.2896L211.024 149.912C221.959 160.846 239.687 160.846 250.622 149.912C261.556 138.977 261.556 121.248 250.622 110.314L153.246 12.9376C152.213 11.3649 151.005 9.87317 149.622 8.49023C144.05 2.91797 136.713 0.185425 129.41 0.292358C122.108 0.185913 114.772 2.91858 109.201 8.49036C107.818 9.87317 106.61 11.3646 105.577 12.9373L8.20093 110.314ZM354.579 375.692V28.2893C354.579 12.8253 367.114 0.289307 382.579 0.289307C398.043 0.289307 410.579 12.8253 410.579 28.2893V375.71L464.201 322.088C475.135 311.154 492.864 311.154 503.799 322.088C514.733 333.023 514.733 350.752 503.799 361.686L406.423 459.062C405.389 460.635 404.181 462.127 402.799 463.51C397.226 469.082 389.889 471.815 382.587 471.708C375.285 471.814 367.949 469.081 362.378 463.51C360.995 462.127 359.787 460.635 358.754 459.063L261.378 361.686C250.443 350.752 250.443 333.023 261.378 322.088C272.312 311.154 290.041 311.154 300.975 322.088L354.579 375.692Z' /%3E%3C/svg%3E");
+    mask-size: contain;
+    mask-position: center;
+    mask-repeat: no-repeat;
+    font-size: inherit;
+  }
+
+  // sort asc
+.lu-action-sort[data-sort$=sc]::before {
```

```suggestion
.lu-action-sort[data-sort='asc']::before {
```

Why don't you write `asc`?

---

I tested this change in Ordino and it works with `.lu-action-sort[data-sort='asc']::before`.

---

I actually had problems with the neutral icon doing so last week when I used asc, but I just tested it again and it seems to work. I'll commit your changes :+1: